accesslog/schema/access_log_record.go

```diff
@@ -97,23 +97,27 @@ func (b *recordBuffer) WriteDashOrStringValue(s string) {
 
 // AccessLogRecord represents a single access log line
 type AccessLogRecord struct {
-	Request                *http.Request
-	HeadersOverride        http.Header
-	StatusCode             int
-	RouteEndpoint          *route.Endpoint
-	RoundtripStartedAt     time.Time
-	FirstByteAt            time.Time
-	RoundtripFinishedAt    time.Time
-	AppRequestStartedAt    time.Time
-	AppRequestFinishedAt   time.Time
-	BodyBytesSent          int
-	RequestBytesReceived   int
-	ExtraHeadersToLog      []string
-	DisableXFFLogging      bool
-	DisableSourceIPLogging bool
-	RedactQueryParams      string
-	RouterError            string
-	record                 []byte
+	Request                    *http.Request
+	HeadersOverride            http.Header
+	StatusCode                 int
+	RouteEndpoint              *route.Endpoint
+	RoundtripStartedAt         time.Time
+	FirstByteAt                time.Time
+	RoundtripFinishedAt        time.Time
+	AppRequestStartedAt        time.Time
+	AppRequestFinishedAt       time.Time
+	BodyBytesSent              int
+	RequestBytesReceived       int
+	ExtraHeadersToLog          []string
+	DisableXFFLogging          bool
+	DisableSourceIPLogging     bool
+	RedactQueryParams          string
+	RouterError                string
+	LogAttemptsDetails         bool
```

Comment:
didn't we say we wanted one `AttemptsDetails` field here that could later be extended?
we are now adding 5 single fields plus one additional flag.

Reply:
Discussed offline: we will leave it like this for now but re-consider this overall structure later.